### DISCUSSION TITLE
Daily Standup - Monday, April 17

### DISCUSSION THREAD

**Discussion by antonieto** [antonieto/ibm-dashboard#130] [General]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:

```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
```

**Comment by antonieto** [comment]
** What did you achieve in the last 24 hours?**:
 - I added an ESLont workflow to enforce code style compliance in every PR

**What are your priorities for the next 24 hours?**:
 - Write the tasks to achieve the DB deployment milestone

**Blockers**:
 - None

**Shoutouts**:
 - @Int-Angel for finishing the DB design

**Comment by Jhoscar22** [comment]
**What did you achieve in the last 24 hours?**:
 - I reviewed some SQL concepts and made some exercises in W3.

**What are your priorities for the next 24 hours?**:
 - Divide the auth into tasks

**Blockers**:
 - None

**Shoutouts**:
 - @Int-Angel  for his fast progress in the DB.
